projects/libyaml/libyaml_emitter_fuzzer.c

```diff
@@ -318,5 +320,9 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     yaml_event_delete(events + k);
```

Comment:
add a early exit label (e.g. error) before the loop over the events, so everything is properly cleaned up
(and instead of returning before, goto error (or similar) )